cmd/kubeadm/app/preflight/checks.go

```diff
@@ -829,14 +829,15 @@ func (ImagePullCheck) Name() string {
 // Check pulls images required by kubeadm. This is a mutating check
 func (ipc ImagePullCheck) Check() (warnings, errors []error) {
 	for _, image := range ipc.imageList {
-		glog.V(1).Infoln("pulling ", image)
 		ret, err := ipc.runtime.ImageExists(image)
 		if ret && err == nil {
+			glog.V(1).Infof("image exists: %s", image)
 			continue
 		}
 		if err != nil {
 			errors = append(errors, fmt.Errorf("failed to check if image %s exists: %v", image, err))
```

Comment:
this still would be an indication that  `inspect` fails for some odd reason.
so nice to have IMHO, even if `pull` inspects existing images, already.